doc/docs/FAQ.md

```diff
@@ -168,7 +168,7 @@ By default, when Meep assigns a dielectric constant ε or μ to each pixel, it u
 
 Still, there are times when, for whatever reason, you might not want this feature. For example, if your accuracy is limited by other issues, or if you want to skip the wait at the beginning of the simulation for it do to the averaging. In this case, you can disable the subpixel averaging by setting `Simulation.eps_averaging = False` (Python) or `(set! eps-averaging? false)` (Scheme). For more details, see [Python User Interface](Python_User_Interface.md).
 
-Even if you disable the subpixel averaging, however, when you output the dielectric function to a file and visualize it, you may notice that there are some pixels with intermediate ε values, right at the boundary between two materials. This is due to a completely different reason. Internally, Meep's simulation is performed on a [Yee grid](Yee_Lattice.md), in which every field component is stored on a slightly different grid which are offset from one another by half-pixels, and the ε values are also stored on this Yee grid. For output purposes, however, it is more user-friendly to output all fields etcetera on the same grid at the center of each pixel, so all quantities are interpolated onto this grid for output. Therefore, even though the internal ε values are indeed discontinuous when you disable subpixel averaging, the output file will still contain some "averaged" values at interfaces due to the interpolation from the Yee grid to the center-pixel grid.
+Even if you disable the subpixel averaging, however, when you output the dielectric function to a file and visualize it, you may notice that there are some pixels with intermediate ε values, right at the boundary between two materials. This is due to a completely different reason. Internally, Meep's simulation is performed on a [Yee grid](Yee_Lattice.md), in which every field component is stored on a slightly different grid which are offset from one another by half-pixels, and the ε values are also stored on this Yee grid. For output purposes, however, it is more user-friendly to output all fields etcetera on the same grid at the center of each pixel, so all quantities are interpolated onto this grid for output. Therefore, even though the internal ε values are indeed discontinuous when you disable subpixel averaging, the output file will still contain some "averaged" values at interfaces due to the interpolation from the Yee grid to the center-pixel grid. These interpolated pixels will conspiculously show when geometries intersect the simulation's `size` boundaries and the `k_point` is set. The set of interpolated values includes values from the Yee grid that has "wrapped around", even when PML is used along some boundary. This is futher explained in [How does `k_point` define the phase relation between adjacent unit cells?](#How-does-kpoint-define-the-phase-relation-between-adjacent-unit-cells)
```

Comment:
As a suggestion, here is an alternative for the added text which provides additional detail: "Also, if `k_point` is set and the boundaries are periodic, the permittivity function of the entire cell obtained via `get_epsilon` or `output_epsilon` will show that a little of the cell from one edge "leaks" over to the other edge. This feature is independent of PML and the way the structure is defined (i.e., using geometric objects or a material function, etc)."

Reply:
I agree. I can work this clarification in, but please feel free to edit it how you see fit. I just wanted to get this jump-started.